pyproject.toml

```diff
@@ -44,6 +44,7 @@ dev = [
     "flake8 == 4.0.1",
     "flake8-bugbear == 22.9.23",
     "mypy == 0.981",
+    "rich == 12.6.0",
```

Comment:
Without rich being listed here, mypy type checks fail because mypy doesn't know how to handle the rich import. 😐 I figured it would at least help ensure folks developing/testing get the nicer output.

Reply:
Right, so there are two ways forward IMO:
- just eat the dependency and add `rich >= 12` to `dependencies`
- make a new `optional-dependencies` section like so:

```
pretty = [
    "rich >= 12",
]
```
Which will allow people to install `rich` support using `fixit[pretty]`, and then in [`tool.hatch.envs.default.features`](https://github.com/Instagram/Fixit/blob/bddbee6b2c06efcb817551155757950c8090399d/pyproject.toml#L69) add `pretty` to make it part of the default environment (which mypy also uses).